content/03.methods.md

```diff
@@ -28,14 +28,16 @@ PNOC collected blood and tumor biospecimens from newly-diagnosed DIPG patients a
 #### PNOC samples
 The Translational Genomic Research Institute (TGEN; Phoenix, AZ) performed DNA and RNA extractions on tumor biopsies using a DNA/RNA AllPrep Kit (Qiagen, #80204). 
 All RNA used for library prep had a minimum RIN of 7 but no QC thresholds were implemented for the DNA.
-For library preparation, 500ng of nucleic acids were used as input for RNA-Seq and WXS.
+For library preparation, 500ng of nucleic acids were used as input for RNA-Seq, WXS, and targeted DNA panel (panel).
 The RNA prep was performed using the TruSeq RNA Sample Prep Kit (Illumina, #FC-122-1001) and the exome prep was performed using KAPA Library Preparation Kit (Kapa Biosystems, #KK8201) using Agilent's SureSelect Human All Exon V5 backbone with custom probes. 
-These probes include CGH probes that target 44,000 evenly spaced genomic loci to assess copy number changes, as well as probes that tile across tumor suppressor genes and genes involved in common cancer translocations.
+The targeted DNA panel developed by Ashion (GEM Cancer panel) consisted of exonic probes against 541 cancer genes.
```

Comment:
Do you think it is worth including a URL for the GEM cancer panel? I took a quick look at the Ashion website and could not identify what citation, if any, would be appropriate.

Reply:
Let me add "formerly known as" to this - I looked as well and it looks like it is old technology that perhaps TGEN now uses. I was a bit confused as well. In other good news, I got permission to share their bed file (and we will rerun our analyses with this bed file as opposed to the WXS file that was used before).